index.js

```diff
@@ -0,0 +1,107 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let form1 = document.querySelector("#form1");
+    let form2 = document.querySelector("#form2");
+    let form3 = document.querySelector("#form3");
+    let addClassName = document.querySelector("#class");
+    let addTeacher = document.querySelector("#teacher");
+    let studentClass = document.querySelector("#studentClass");
+    let studentName = document.querySelector("#studentName");
+    let studentAge = document.querySelector("#studentAge");
+    let studentCity = document.querySelector("#studentCity");
+    let studentGrade = document.querySelector("#studentGrade");
+    let passingClass = document.querySelector("#passingClass");
+    let City = document.querySelector("#city");
```

Comment:
Be careful with naming your variables . Capitalizing is usually reserved for class, components, etc.